index.html

```diff
@@ -182,6 +182,10 @@
           be assigned the new type and the interaction model of the resource MUST be changed to the
           interaction model assigned to the new type by [[LDP]].
         </p>
+        <p>
+          A PUT request that creates a new resource MUST NOT, while so doing, change the containment
+          triples in any LDPC. ([[LDP]] 5.2.4.1 SHOULD becomes MUST)
```

Comment:
As I understand ReSpec (which is somewhat limited), `[[BLAH]]` means "informative" while `[[!BLAH]]` means normative. We're clearly inconsistent in the document, but in this case, I think we want `[[!LDP]]`.